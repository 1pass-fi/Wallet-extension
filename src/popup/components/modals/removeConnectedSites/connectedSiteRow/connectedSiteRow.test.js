import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import React from 'react'

import ConnectedSiteRow from '.'

describe('Test for ConnectedSiteRow component', () => {
  describe('Render without crashing', () => {
    it('renders correctly', () => {
      const { container } = render(<ConnectedSiteRow />)
      expect(container).toMatchSnapshot()
    })
  })
})
