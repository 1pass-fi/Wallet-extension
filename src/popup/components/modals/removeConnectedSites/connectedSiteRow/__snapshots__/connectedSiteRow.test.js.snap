// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for ConnectedSiteRow component Render without crashing renders correctly 1`] = `
<div>
  <div
    class="connected-site-row"
    style="background: rgb(255, 255, 255);"
  >
    <div
      class="connected-site-name"
    />
    <button
      class="delete-button"
    >
      <div
        class="delete-icon"
      >
        <svg />
      </div>
    </button>
  </div>
</div>
`;
